### DISCUSSION TITLE
Table or Grid?

### DISCUSSION THREAD

**Discussion by mfreeman-xtivia** [lowdefy/lowdefy#1175] [Questions and Help] [answered]
Maybe I missed it, but from a quick scan of the docs I do not see any component that can be used as a grid or table to represent tabular rows of data?

Is there such a beast?

**Comment by Gervwyk** [comment, marked as answer]
Hi @mfreeman-xtivia 
Yes, we generally use ag-grid to render tables. See https://github.com/lowdefy/blocks-aggrid
It is currently not in the docs because it is not a default type. This might change in the future, but for now, you need to add the type definition in your app. See the examples: https://github.com/lowdefy/blocks-aggrid#aggridalpine-example

**Comment by mfreeman-xtivia** [comment]
Do you know if ag-grid is licensed the same way as lowdefy?

**Reply by SamTolmay** [reply]
The Lowdefy AgGrid block uses the community version, which is available under the MIT license: https://www.ag-grid.com/license-pricing.php
